packages/ckeditor5-font/src/fontsize/fontsizeui.ts

```diff
@@ -76,6 +76,19 @@ export default class FontSizeUI extends Plugin {
 				editor.editing.view.focus();
 			} );
 
+			// Show label on dropdown's button.
+			dropdownView.buttonView.set( 'withText', true );
+
+			// Hide the icon.
+			dropdownView.buttonView.set( 'icon', false );
+
+			// Bind dropdown's button label to fontSize value.
+			dropdownView.buttonView.bind( 'label' ).to( command, 'value', value => {
+				// If no value is set on the command show default size.
+				// Use t() method to make that string translatable.
+				return value ? value : t( '16px' );
+			} );
+
```

Comment:
Hi @gvgesanto2,

I think we don't need to change anything in this file, Do you agree?

Reply:
Hi @Bondok6, you are right! Thank you for the catch! I just fixed it!